docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -1,13 +1,105 @@
 # Dasharo Compatibility: Network Boot
 
-## Test cases
+## PXE001.001 Dasharo Network Boot is available
 
-### PXE001.001 iPXE network boot
+**Test description**
+
+This test aims to verify that the `iPXE Network boot` option is available and
+whether you are entered to `Dasharo Network Boot Menu` after selecting it.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
```

Comment:
`Test setup` stages for all test cases in this suite should contain two points:

1. Proceed with the [Generic test setup: firmware](../../generic-test-setup#firmware).
2. Device should be connected to the Internet by using an Ethernet cable.

As all of these sections should be identical, I think that we should create `common section`.

Reply:
applied